avocado/core/utils/messages.py

```diff
@@ -141,121 +136,11 @@ def get(cls, msg, path):  # pylint: disable=W0221
         return super().get(msg=msg, path=path)
 
 
-_supported_types = {
+SUPPORTED_TYPES = {
     LogMessage.message_type: LogMessage,
     StdoutMessage.message_type: StdoutMessage,
     StderrMessage.message_type: StderrMessage,
     WhiteboardMessage.message_type: WhiteboardMessage,
     OutputMessage.message_type: OutputMessage,
     FileMessage.message_type: FileMessage,
 }
-
-
-class RunnerLogHandler(logging.Handler):
-    def __init__(self, queue, message_type, kwargs=None):
-        """
-        Runner logger which will put every log to the runner queue
-
-        :param queue: queue for the runner messages
-        :type queue: multiprocessing.SimpleQueue
-        :param message_type: type of the log
-        :type message_type: string
-        """
-        super().__init__()
-        self.queue = queue
-        self.message = _supported_types[message_type]
-        self.kwargs = kwargs or {}
-
-    def emit(self, record):
-        msg = self.format(record)
-        self.queue.put(self.message.get(msg, **self.kwargs))
-
-
-class StreamToQueue:
-    def __init__(self, queue, message_type):
-        """
-        Runner Stream which will transfer data to the runner queue
-
-        :param queue: queue for the runner messages
-        :type queue: multiprocessing.SimpleQueue
-        :param message_type: type of the log
-        :type message_type: string
-        """
-        self.queue = queue
-        self.message = _supported_types[message_type]
-
-    def write(self, buf):
-        self.queue.put(self.message.get(buf))
-
-    def flush(self):
-        pass
-
-
-def start_logging(config, queue):
```

Comment:
This is not avoca-instrumented runner specific method and it is used by other runners too, for example [avocado-vt](https://github.com/avocado-framework/avocado-vt/blob/master/avocado_vt/plugins/vt_runner.py#L65). Because of this, it can't be removed from `utils.messages`.

Reply:
You are right.  I've checked that `RunnerLogHandler` and `StreamToQueue` were not used in Avocado-VT, but I forgot about `start_logging()`.